access_application_test.go

```diff
@@ -455,3 +455,79 @@ func TestAccessApplicationWithCORS(t *testing.T) {
 		assert.Equal(t, want, actual)
 	}
 }
+
+func TestCreatePrivateAccessApplication(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodPost, r.Method, "Expected method 'POST', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, `{
+			"success": true,
+			"errors": [],
+			"messages": [],
+			"result": {
+				"id": "480f4f69-1a28-4fdd-9240-1ed29f0ac1db",
+				"created_at": "2014-01-01T05:20:00.12345Z",
+				"updated_at": "2014-01-01T05:20:00.12345Z",
+				"aud": "737646a56ab1df6ec9bddc7e5ca84eaf3b0768850f3ffb5d74f1534911fe3893",
+				"name": "Private Admin Site",
+				"private_address": "198.51.100.0",
+				"type": "private_ip",
+		    "gateway_rules": [
+					{"id": "d9c61460-6f4d-4c40-89ea-2f552c9a8466"},
+					{"id": "bc5ee7e7-9773-47a3-835e-b9b9799ebb92"}
+				],
+				"session_duration": "24h",
+				"allowed_idps": ["f174e90a-fafe-4643-bbbc-4a0ed4fc8415"],
+				"auto_redirect_to_identity": false,
+				"enable_binding_cookie": false,
+				"custom_deny_url": "https://www.cloudflare.com",
```

Comment:
where we are using a URL in tests, can you please swap to `example.com`?

Reply:
Updated!